src/components/ProfilePage.tsx

```diff
@@ -153,6 +161,33 @@ export const ProfilePage = (props: ProfilePageProps) => {
         </>
     ) 
 
+    const onCropComplete = (croppedArea: any, croppedAreaPixels: any) => {
+		setCroppedArea(croppedAreaPixels);
+	};
+
+    const onCrop = async() => {
+        const file: any = await getCroppedImg(image!, croppedArea!)
+        try{
+            setLoadImg(true)
+            const resp: FileUploadComplete = await uploadFile(accessToken, file.name, file)
+            const picUrl = await getFileHandleByIdURL(resp.fileHandleId,accessToken)
+            setFileHandleId(resp.fileHandleId)
+            setProfilePicUrl(picUrl)
+        } catch (err:any){
+            displayToast(err.reason as string,'danger')
+        }
+        setLoadImg(true)
+        closeCropModal()
+    }
```

Comment:
Creates a new file with the cropped image and uploads file. Then replaces the same file into the original profile picture.